apis/python/src/tiledbsoma/sparse_nd_array.py

```diff
@@ -374,3 +301,68 @@ def tables(self) -> TableReadIter:
         [lifecycle: experimental]
         """
         return TableReadIter(self.sr)
+
+
+def _build_tiledb_schema(
```

Comment:
super tiny nit: given that this is used by *NDArray, you might consider moving it into util_tiledb.py - it threw me when I saw the DenseNDArray implementation importing it :-)